### ISSUE TITLE
bug: Updating tools hangs and results 500 error

### ISSUE THREAD

**Issue by PekkaJalonen** [yogeshojha/rengine#1152]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current Behavior

When updating some tools in arsenal, example ctfr will caus web to hang with "Downloading laterst version..." and looking at the container logs, it shows:

django.server | "GET /api/github/tool/get_latest_releases/?tool_id=12 HTTP/1.0" 200 38
run_command                        | INFO | cd /usr/src/github/ctfr && git pull && cd -
run_command                        | WARNING | None
django.server | "GET /api/tool/update/?tool_id=12 HTTP/1.0" 500 133332

### Expected Behavior

Update should respond if the update was successfull or not.

### Steps To Reproduce

1. In tool arsenal, select ctfr and click Check Update
2. Click Force download
3. Web page hangs in "Downloading laterst version..."

### Environment

```markdown
- reNgine: 2.0.2 (fresh install)
- OS: Ubuntu 22.04
- Python: 3.10.12
- Docker Engine: 24.0.6
- Docker Compose: 2.21.0
- Browser: Microsoft Edge 120.0.2210.61
```

**Comment by AnonymousWP** [comment]
I don't think I had/have this issue. Can you still reproduce it?

**Comment by psyray** [comment]
I can reproduce
![image](https://github.com/yogeshojha/rengine/assets/1230954/fb735b59-f2a4-4404-ad14-2101badbe425)

I will have a look

**Comment by yogeshojha** [comment]
@PekkaJalonen 

Thank you for reporting this.

A PR has been created to fix this, this has been fixed with PR #1335 

Thank you
